### DISCUSSION TITLE
Col-auto doesn't have width equal to its content when content is a nested BS grid

### DISCUSSION THREAD

**Discussion by VMLuca** [twbs/bootstrap#40614] [Q&A]
I'm trying to make col-auto to have width equal to its content as described in [Auto-sizing](https://getbootstrap.com/docs/5.3/forms/layout/#auto-sizing)

In my code snippet you can see I have a col-auto which has a nested grid for its children. The problem is that col-auto container's width is larger then the content of it's children. I expect it to have width equal to text in Latin.

I am looking forward to any advice or hint!

### Reduced test cases

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link
      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css"
      rel="stylesheet"
      integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH"
      crossorigin="anonymous"
    />
  </head>
  <body>
    <div class="container">
      <div class="row">
        <div class="col-auto">
          <div class="row">
            <div class="col-12 border">
              Lorem ipsum dolor sit amet consectetur adipisicing elit.
            </div>
            <div class="col-12 border">
              Lorem ipsum dolor sit amet consectetur adipisicing elit.
            </div>
          </div>
        </div>
        <div class="col border" style="aspect-ratio: 1/1">2 of 2</div>
      </div>
    </div>
  </body>
</html>
```

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Bootstrap are you using?

5.3.3

**Comment by VMLuca** [comment]
@julien-deramond are you sure this is not a bug?

**Reply by julien-deramond** [reply]
No, I'm not. The description seemed more like a Q&A discussion (e.g., 'I am looking forward to any advice or hint!'). However, once someone from the community reviews it, we can still create or recreate an issue if it turns out to be a bug :)
